src/Microsoft.IdentityModel.Tokens/AppContextSwitches.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.IdentityModel.Tokens
+{
+    /// <summary>
+    /// AppContext switches for Microsoft.IdentityModel.Tokens and referencing packages.
+    /// </summary>
+    public static class AppContextSwitches
```

Comment:
Either the class should also be internal or the consts should also be public

Reply:
Is there no option to use this switch in the user code since it is internal now?
The only remaining way to use it is to pass the value explicitly, like this:
```csharp
AppContext.SetSwitch("Microsoft.IdentityModel.Tokens.UseClaimsIdentityType", true);
```

Reply:
Yes, that was the current behavior that we didn't change. They're internal to avoid new dependencies on them so that eventually the switches can be removed in later versions. This class is meant to only be used internally for ease of use as long as the switch is needed, and users can reference this file for what switches are supported.